main.go

```diff
@@ -86,6 +86,10 @@ func main() {
 		http.HandleFunc(*metrics, metricHandler(services))
 	}
 
+	if *bind == "" {
+		log.Fatal("--web.listen-address is empty. Please specify address:port to listen on.")
```

Comment:
Should be a default value, rather than a log fatal

Reply:
last commit should fix it